client/state/ui/actions.js

```diff
@@ -89,3 +90,11 @@ export const toggleNotificationsPanel = () => {
  * @return {Object}      Action object
  */
 export const navigate = path => ( { type: NAVIGATE, path } );
+
+/**
+ * Hide or show the masterbar.
+ *
+ * @param {bool} hide True to hide masterbar
+ * @return {Object} Action object
+ */
+export const setHideMasterbar = hide => ( { type: MASTERBAR_HIDE, hide } );
```

Comment:
sorry for the duplicate comments - just trying to knit together the garment, so to speak…

```js
export const hideMasterbar = () => ( { type: MASTERBAR_TOGGLE_VISIBILITY, isVisible: false } )

export const showMasterbar = () => ( { type: MASTERBAR_TOGGLE_VISIBILITY, isVisible: true } )
```

Reply:
"toggle" always makes me think of something that switches between two states on its own, like a "nullary" action. I prefer "set" when we provide the value, which we should be doing:

Given that this action name is behind nicely explicit action creators for the two states, this is very much a nitpick and doesn't need to be changed 🙂